bundles/python/python_mode.moon

```diff
@@ -0,0 +1,34 @@
+-- Copyright 2012-2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+{
+  lexer: bundle_load('python_lexer')
+
+  comment_syntax: '#'
+
+  default_config:
+    word_pattern: r'\\b[\\w_][\\w\\d_]+\\b'
```

Comment:
Ah, good catch. This is also a big problem for the lexer then? `alpha` wont match all unicode letters. It also seems to depend on the current locale which would make the behavior inconsistent.